app/src/main/java/com/hedvig/app/feature/keygear/KeyGearValuationActivity.kt

```diff
@@ -1,23 +1,39 @@
 package com.hedvig.app.feature.keygear
 
+import android.content.Context
+import android.content.Intent
+import android.content.res.ColorStateList
 import android.os.Bundle
+import com.hedvig.android.owldroid.type.MonetaryAmountV2Input
 import com.hedvig.app.BaseActivity
 import com.hedvig.app.R
-import com.hedvig.app.feature.keygear.ui.itemdetail.KeyGearValuationViewModelImpl
+import com.hedvig.app.feature.keygear.ui.itemdetail.KeyGearItemDetailViewModel
 import com.hedvig.app.feature.keygear.ui.itemdetail.PurchaseDateYearMonthPicker
+import com.hedvig.app.util.extensions.compatColor
 import com.hedvig.app.util.extensions.observe
+import com.hedvig.app.util.extensions.onChange
 import com.hedvig.app.util.extensions.view.setHapticClickListener
 import kotlinx.android.synthetic.main.activity_key_gear_valuation.*
 import org.koin.android.viewmodel.ext.android.viewModel
+import org.threeten.bp.YearMonth
+import timber.log.Timber
 import java.text.DateFormatSymbols
 
-class KeyGearValuationActivity : BaseActivity(R.layout.activity_key_gear_valuation) {
+class KeyGearValuationActivity :
+    BaseActivity(R.layout.activity_key_gear_valuation) {
 
-    private val model: KeyGearValuationViewModelImpl by viewModel()
+    private val model: KeyGearItemDetailViewModel by viewModel()
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
+        val id = intent.getStringExtra(ITEM_ID)
+        var date: YearMonth? = null
+
+        continueButton.isEnabled = false
+        continueButton.backgroundTintList =
```

Comment:
I think we should take the Hedvig Button style to the next level and use a Color state list with a disabled-selector:

https://developer.android.com/guide/topics/resources/color-list-resource